### PR TITLE
Add block GQL endpoint

### PR DESCRIPTION
**Pull request by trevorbernard**

### CODE REVIEW

**Comment by jhult** [comment]
Implements #553

File: rust/src/web/mod.rs [hunk 1]
```diff
@@ -31,6 +37,16 @@ pub async fn start_web_server<A: net::ToSocketAddrs, P: AsRef<Path>>(
...
+            .service(
+                web::resource("/graphql")
+                    .guard(guard::Post())
+                    .to(GraphQL::new(schema.clone())),
+            )
+            .service(
+                web::resource("/graphql")
+                    .guard(guard::Get())
+                    .to(index_graphiql),
+            )
```
Comment (jhult) [inline rust/src/web/mod.rs:49]: Should we inline the schema variable?

image

File: rust/src/web/mod.rs [hunk 2]
```diff
@@ -31,6 +37,16 @@ pub async fn start_web_server<A: net::ToSocketAddrs, P: AsRef<Path>>(
             .service(blocks::get_block)
             .service(accounts::get_account)
             .service(blockchain::get_blockchain_summary)
+            .service(
+                web::resource("/graphql")
+                    .guard(guard::Post())
+                    .to(GraphQL::new(schema.clone())),
+            )
+            .service(
+                web::resource("/graphql")
```
Comment (jhult) [inline rust/src/web/mod.rs:46]: Should we make "/graphql" a `const`?

Example: `pub const ENDPOINT_GRAPHQL: &str = "/graphql";`

**Review by jhult — COMMENTED** [review]
Nice work!

**Review by Isaac-DeFrain — APPROVED** [review]
LGTM

**Pull request merged**